### DISCUSSION TITLE
How to keep the current working directory when split tabs

### DISCUSSION THREAD

**Discussion by Creeki** [Eugeny/tabby#8076] [Q&A]
Hi everyone!

When I split current tabs, in the new tab, the working directory would be initialed to my home `~`

Is there any method to split the tabs and guides into the same working directory?

Many thanks!

**Comment by NRKWL** [comment]
This is not happening to me 
```
MacOS λ ./Tabby --version
1.0.196
```
I dont see a setting in Tabby. Please verify your shell profile (like .bashrc or .bash_profile) is not running `cd`.
